pkg/test/framework/components/istio/eastwest.go

```diff
@@ -74,7 +74,7 @@ func (i *operatorComponent) deployEastWestGateway(cluster resource.Cluster) erro
 	}
 
 	installSettings := []string{
-		"manifest", "generate",
+		"install",
```

Comment:
Rather than using install directly here, there is a helper function named `install` that does a preliminary `generate` to save the generated manifest for cleanup at the end of the test.

Reply:
Thanks for the suggestion!!